src/views/guidelines/guidelines.scss

```diff
@@ -0,0 +1,11 @@
+@import "../../colors";
+@import "../../typography";
+
+.view {
+    .guidelines-footer {
+        text-align: center;
```

Comment:
Ah I I think I see – `.view` should actually be `#view` in this case (it's an `id` in the template). Changing that works for me – and I think that the `text-align: center` is sufficient, so I think you could remove the `img` formatting below